dataeng/jobs/analytics/SnowflakeQueueDepth.groovy

```diff
@@ -0,0 +1,62 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeQueueDepth {
+"""
+This job runs every 5 minutes using the Snowflake task automation user.
+"""
+    public static def job = { dslFactory, allVars ->
+
+        dslFactory.job("collect-queue-depth") {
+
+            logRotator common_log_rotator(allVars)
+            parameters secure_scm_parameters(allVars)
+            parameters {
+                stringParam('ANALYTICS_TOOLS_URL', allVars.get('TOOLS_REPO_URL'), 'URL for the analytics tools repo.')
+                stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analytics tools repo to use.')
+            }
+            environmentVariables {
+                env('KEY_PATH', allVars.get('KEY_PATH'))
+                env('PASSPHRASE_PATH', allVars.get('PASSPHRASE_PATH'))
+                env('SNOWFLAKE_USER', allVars.get('SNOWFLAKE_USER'))
+                env('SNOWFLAKE_ACCOUNT', allVars.get('SNOWFLAKE_ACCOUNT'))
```

Comment:
These two parameters are no longer used, so it's effectively dead code.  Instead of deleting the lines, I would actually recommend changing them to 
```suggestion
                env('SNOWFLAKE_USER', 'SNOWFLAKE_TASK_AUTOMATION_USER')
                env('SNOWFLAKE_ACCOUNT', 'edx.us-east-1')
```
and change the bash script to use these environment variables like before.  The reason is that we still want to retain the flexibility of being able to manually override these values while running the job.